draft-ietf-quic-transport.md

```diff
@@ -2679,7 +2679,9 @@ endpoint receives.
 This design relies on the peer always sending a connection ID in its packets so
 that the endpoint can use the connection ID from a packet to reset the
 connection.  An endpoint that uses this design cannot allow its peers to send
-packets with a zero-length destination connection ID.
+packets with a zero-length destination connection ID and need to either use a
```

Comment:
"use ~a~ the same"

Reply:
need -> needs

Reply:
Good catch.  I was reading "packets ... need", but it's actually "An endpoint ... needs"